product-scenarios/pom.xml

```diff
@@ -75,7 +75,7 @@
     <modules>
         <module>scenarios-commons</module>
         <module>1-integrating-systems-that-communicate-in-heterogeneous-message-formats</module>
-        <module>2-Bridging-systems-that-communicate-in-different-protocols</module>
+        <!--<module>2-Bridging-systems-that-communicate-in-different-protocols</module>-->
```

Comment:
I suppose this need not be commented out

Reply:
This is being modified to suite new conventions by Kasun. Until then keeping commented to make sure build passes.

Reply:
AFAIK the build currently passes with module 2

Reply:
Correct. But this is to avoid possible conflicts. He will modify to suit new requirements and enable it.